### PR TITLE
Rebuild lighttpd 1.4.51 with libxml2 2.10

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Actions:
1. Sort dependencies
2. Pin libxml2 to 2.10
3. conda-forge build this package for openssl 1.1.1 and 3.0 so I pin this recipe to 1.1.1
4. Fix home and doc urls, license name
5. Add license_family

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -6,31 +6,33 @@ package:
...
   build:
-    - pkg-config
-    - libtool
-    - perl  # Needed to run the tests
     - {{ compiler('c') }}
-    - make  # [unix]
     - automake
+    - libtool
+    - make  # [unix]
+    - perl  # Needed to run the tests
```
Comment (akabanovs) [inline recipe/meta.yaml:24]: if perl is only required to run the tests, should not we move it to the requirements under tests section? If we decide to run tests using --test argument (i.e. not build but test existing package), will it sill work if perl is inside build section and not tests section?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -41,8 +43,9 @@ test:
     - lighttpd -t -f lighttpd.conf
 
 about:
-  home: http://www.lighttpd.net/
-  license: BSD 3-Clause
+  home: https://www.lighttpd.net/
+  license: BSD-3-Clause
+  license_family: BSD
   license_file: COPYING
```
Comment (akabanovs) [inline recipe/meta.yaml:49]: Should we update the licence file? The correct name is Copyright

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:24]: But tests are running by `make check`, see https://github.com/AnacondaRecipes/lighttpd-feedstock/blob/7d7e5fd2795bc9acaaabd43a2bc74f515c2a05f6/recipe/build.sh#L31-L33 and https://redmine.lighttpd.net/projects/lighttpd/wiki/RunningUnitTests

File: recipe/meta.yaml (hunk 2 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:49]: COPYING is the correct file for v1.4 https://github.com/lighttpd/lighttpd1.4/blob/master/COPYING

File: recipe/meta.yaml (hunk 1 above)
Comment (akabanovs) [reply recipe/meta.yaml:24]: ah, makes sense.  Thanks!

**Review by akabanovs — APPROVED** [review]
LGTM

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**